[WIP] #4448 - Fail early if output engine file path is unwritable in trtexec

PierreMarieCurie opened the pull request:
Hi NVIDIA team,

This PR adds an early check in trtexec to ensure the --saveEngine path is writable before starting ONNX parsing. It avoids unnecessary compute when the path is invalid.

Thanks!

poweiw commented:
/blossom-ci

poweiw commented:
/blossom-ci

poweiw commented:
/blossom-ci

poweiw commented:
Hello @PierreMarieCurie thanks for your contribution! Can you target the main branch and have your commits rebase on that?

PierreMarieCurie commented:
Hello @poweiw, thanks for your feedback. I've rebased my branch onto main and updated the PR to target it. Let me know if there's anything else you'd like adjusted.

PierreMarieCurie commented:
Hello @poweiw and @kevinch-nv, I messed up my rebase, sorry about that. The target branch was correct, but it was based on an old commit, so the pipeline failed. It should be fixed now.

PierreMarieCurie commented:
Hello, any update on this MR when you get a chance? Thanks a lot!

wondfor commented:
/blossom-ci
